src/main/webapp/app/admin/lti-configuration/lti-configuration.component.ts

```diff
@@ -62,7 +62,7 @@ export class LtiConfigurationComponent implements OnInit {
      * Gets the deep linking url
      */
     getDeepLinkingUrl(): string {
-        return `${location.origin}/lti/deep-linking`; // Needs to match url in CustomLti13Configurer
+        return `${location.origin}/api/public/lti13/deep-link`; // Needs to match url in CustomLti13Configurer
```

Comment:
I'd recommend refactoring these hard-coded values into (potentially static) variables that other parts of the code read from and compose URLs based on (similar to server-side code). `'lti13/...'` is a recurring constant in this particular component, and these implicit couplings are prone to errors (someone might forget updating one value while updating another, for example).